src/main/java/org/apache/mesos/scheduler/repair/constrain/TestingLaunchConstrainer.java

```diff
@@ -0,0 +1,31 @@
+package org.apache.mesos.scheduler.repair.constrain;
```

Comment:
I'd be happy to move this; however, the 2 testing classes (for Constrainer and Monitor) are designed to make it easy to write tests for RepairScheduler integration. If these are moved to the `test` tree, the won't be included in the library, and so they couldn't be used for testing, which defeats the point. If you'd like, I can delete these helpers for testing consumers of this library.